src/components/button/button.js

```diff
@@ -0,0 +1,16 @@
+// import Button from '@mui/material/Button';
+
+// const SuccessButton = (props) => {
+//   const {type} = props;
+
+//   const handleClick = () => {
+//     if(type === 'search') {
+
+//     };
+//   };
+//   return (
+//     <Button color="success" variant="outlined" size="small" onClick={handleClick}> Go</Button>
+//    );
+// }
+
+// export default SuccessButton;
```

Comment:
- [ ] Kindly remove this file as it is not being used. This will make your codebase cleaner and more professional